examples/common/lib/utils.sh

```diff
@@ -139,6 +139,31 @@ function wait_for_healthy_shard() {
 	wait_for_shard_vreplication_engine "${keyspace}" "${shard}"
 }
 
+# Stop the specified vtadmin binary name using the provided PID file.
+# Example:
+#  stop_vtadmin_process "vtadmin-web" "$VTDATAROOT/tmp/vtadmin-web.pid"
+function stop_vtadmin_process() {
+  if [[ -z ${1} || -z ${2} ]]; then
+    fail "A binary name and PID file must be specified when attempting to shutdown vtadmin"
```

Comment:
I see `fail` defined below and doing an `exit`, this exits the full bash process, which might not be what we want if there are many stops in a single process.

Reply:
> if there are many stops in a single process

I meant if many things can `fail` in a single bash script and we want to continue to the next thing in the script.